games/fheroes2/Portfile

```diff
@@ -18,9 +18,9 @@ long_description \
 
 homepage        https://ihhub.github.io/fheroes2/
 
-checksums       rmd160  8bd4ff6eb683c790561be087a5c8c894781382a6 \
-                sha256  fc4f3791f061f2c94ee8fb391590bab1de3e283bba5e0d07e03f2299a75eac37 \
-                size    3265126
+checksums       rmd160  df9e2fa1479b7f3a60199785bf937a2ce9ce1c73 \
+                sha256  e70ee6417cf91fa00a99f9ef7e83bf8accc4619979ac0a0ec3687dbd51cf45e2 \
+                size    3277427
 
 depends_build-append \
                 port:gettext
```

Comment:
since you're updating this port anyway, please take care of [Trac ticket 64291](https://trac.macports.org/ticket/64291) as well.

Reply:
Hi @reneeotten this port depends on `gettext` only in order to generate `.pot` file and then `.mo` files from `.po`files during its build process - in other words, it needs only the `xgettext`, `msgmerge` and `msgfmt` binaries, not the runtime (the upstream has its own runtime implementation).